### PR TITLE
Fixes #37070 - BreadcrumbBar.test.js missing await

### PR DESCRIPTION
**Pull request by MariaAga**

### CODE REVIEW

**Review by kmalyjur — APPROVED** [review]
LGTM :+1:

File: webpack/assets/javascripts/react_app/components/BreadcrumbBar/__tests__/__snapshots__/BreadcrumbBar.test.js.snap [hunk 1]
```diff
@@ -110,7 +110,7 @@ Array [
       "switcherItemUrl": "some/url/:id",
     },
     Object {
-      "page": 1,
```
Comment (ofedoren) [inline webpack/assets/javascripts/react_app/components/BreadcrumbBar/__tests__/__snapshots__/BreadcrumbBar.test.js.snap:113]: Just a question, not blocking: is page differences expected? If so, why?

Comment (MariaAga) [reply webpack/assets/javascripts/react_app/components/BreadcrumbBar/__tests__/__snapshots__/BreadcrumbBar.test.js.snap:113]:
```
    rerender(
        <BreadcrumbBar
          {...{ ...props, isSwitcherOpen: true, currentPage: 2, total: 40 }}
        />
      );
``` 
 -> page is 2
```
      await act(async () =>
        fireEvent.click(screen.getByLabelText('Go to next page'))
      );
      expect(props.loadSwitcherResourcesByResource.mock.calls).toHaveLength(2);
```
-> page is 3 in the second call

```
   await act(async () =>
        fireEvent.click(screen.getByLabelText('Go to previous page'))
      );
      expect(props.loadSwitcherResourcesByResource.mock.calls).toHaveLength(3);
```

-> page is 2 in the 3rd call

Comment (ofedoren) [reply webpack/assets/javascripts/react_app/components/BreadcrumbBar/__tests__/__snapshots__/BreadcrumbBar.test.js.snap:113]: Oh, my bad, should've looked at the test better :/ Thank you :)

**Review by ofedoren — APPROVED** [review]

**Pull request merged**